src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -229,6 +229,40 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                 }
             },
 
+            outerHTML: {
+                condition: domUtils.isElementNode,
+                get:       el => cleanUpHtml(el.outerHTML, el.parentNode && el.parentNode.tagName),
```

Comment:
Before call of `get` function, call `condition` which checks that `el` is node element